### PR TITLE
Task/sapnamysoree/tlt 3620/add school info

### PR DESCRIPTION
**Pull request by sapnamysore**
This PR has changes to display account/school info in settings page based on teh bookmarklet written bu Artie. details in this item: https://jira.huit.harvard.edu/browse/TLT-3620

The change is deployed on canvas.dev
https://canvas.dev.tlt.harvard.edu/courses/4993/settings

### CODE REVIEW

File: js/course_settings_add_school_detail.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+function handleError(errorObject) {
+    var errorText = "Error: " + errorObject.message;
+    console.log(errorObject);
+    annotatePage(errorText, {color: "#721c24", backgroundColor: "#f8d7da"}) || alert(errorText);
+    return errorObject;
+}
+
+function fethcAccountInfo() {
```
Comment (Chris-Thornton-Harvard) [inline js/course_settings_add_school_detail.js]: Typo in fetch

File: js/course_settings_add_school_detail.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }
+}
+
+function initAddSchoolDetail() {
+  var reCourseSettingsPage = /courses\/.+?\/settings/;
+  var windowUrl = window.location.pathname;
+  var onCourseSettingsPage = (windowUrl.search(reCourseSettingsPage) != -1);
+
+  if (onCourseSettingsPage) {
+    fethcAccountInfo();
```
Comment (cmurtaugh) [inline js/course_settings_add_school_detail.js]: If the typo above is fixed, it'll need to be fixed here too :-)

**Comment by cmurtaugh** [comment]
Couple minor typos but otherwise looks good!

**Comment by Chris-Thornton-Harvard** [comment]
Redeployed the JS to dev and it looks good!

**Pull request merged**